stubs/uWSGI/uwsgi.pyi

```diff
@@ -0,0 +1,241 @@
+from _typeshed import HasFileno, OptExcInfo, ReadOnlyBuffer
+from _typeshed.wsgi import WSGIApplication
+from collections.abc import Callable
+from types import ModuleType
+from typing import Any, Protocol, overload
+from typing_extensions import Literal, Self, TypeAlias, final
+
+import uwsgidecorators
+
+_TrueOrNone: TypeAlias = Literal[True] | None
+
+class _RPCCallable(Protocol):
+    def __call__(self, *args: bytes) -> bytes | None: ...
+
+# FIXME: Technically we know the exact layout of _AppsDict and _WorkerDict
+#        but TypedDict does not support bytes keys, so for now we use type
+#        aliases to a more generic dict
+_AppsDict: TypeAlias = dict[bytes, int | bytes]
+_WorkerDict: TypeAlias = dict[bytes, int | bytes | tuple[_AppsDict, ...]]
+
+SPOOL_IGNORE: Literal[0]
+SPOOL_OK: Literal[-2]
+SPOOL_RETRY: Literal[-1]
+applications: dict[str, WSGIApplication | str] | None
+buffer_size: int
+cores: int
+has_threads: int
+hostname: bytes
+is_a_reload: bool
+loop: bytes | None
+magic_table: dict[bytes, bytes]
+numproc: int
+opt: dict[str, bytes | Literal[True] | list[bytes | Literal[True]]]
```

Comment:
Is this dict meant to be used for setting options or for getting their values? If it's meant mostly for getting, this too could be better with `Any`. Same goes for the `applications` dict.

Reply:
As far as I can tell it's purely for reading options, yes. I'm fine with changing it, but it seemed helpful to know that the values would be bytes, even though the key is a regular string, which is usually not the case in uWSGI.